src/GraphQL.Tests/Bugs/Bug1781.cs

```diff
@@ -0,0 +1,95 @@
+using System.Numerics;
+using System.Threading.Tasks;
+using GraphQL.Language;
+using GraphQL.Types;
+using GraphQLParser.AST;
+using Shouldly;
+using Xunit;
+
+namespace GraphQL.Tests.Bugs
+{
+    // https://github.com/graphql-dotnet/graphql-dotnet/pulls/1781
+    public class Bug1781 : QueryTestBase<Bug1781Schema>
```

Comment:
This is the way to name the tests that correspond to the bugs that have been fixed. Actually there is no 1781 issue reporting some bug.

Reply:
I propose to rename this file to `CoreToVanillaConverterTests.cs` and move into GraphQL.Tests root folder.

Reply:
👍

Reply:
I will rely on your guidance for how/where to name tests.  So please comment along these lines as you see fit.